google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriter.java

```diff
@@ -679,4 +708,33 @@ public ProcessBulkCommitCallback(BulkCommitBatch batch, int attempt) {
   void setMaxBatchSize(int size) {
     maxBatchSize = size;
   }
+
+  @VisibleForTesting
+  RateLimiter getRateLimiter() {
+    return rateLimiter;
+  }
+
+  private void validateBulkWriterOptions(BulkWriterOptions options) {
+    double initialRate = options.getInitialOpsPerSecond();
+    double maxRate = options.getMaxOpsPerSecond();
+
+    if (initialRate < 1) {
+      throw FirestoreException.invalidState(
+          "Value for argument 'initialOpsPerSecond' must be an integer within [1, Infinity], but was: "
```

Comment:
```suggestion
          "Value for argument 'initialOpsPerSecond' must be greater than 1, but was: "
```

Easier to read, me thinks.